src/layouts/Layout.jsx

```diff
@@ -36,7 +36,9 @@ const Layout = ({ ariaLabel, children }) => {
       <NavBarSkipLink />
       <NavBar />
       {session.info.isLoggedIn && <Breadcrumbs />}
-      <main id="main-content">{children}</main>
+      <div id="main-content" tabIndex={-1}>
```

Comment:
Oops reverting this back to `main`. I had been playing around making this a component on my local.